### PR TITLE
Fea/new header

### PR DESCRIPTION
**Pull request by JonasBa**
New community header added to the build and removed the old unnecessary css and html

### CODE REVIEW

**Comment by bobylito** [comment]
Thanks @JonasBa that's awesome ! 

For the sake of readability can you not push the doc built? :)

**Comment by JonasBa** [comment]
Yes, ofc @bobylito :) Let me remove that :)

**Comment by JonasBa** [comment]
@bobylito should be good now ;)

**Comment by bobylito** [comment]
http://noisy-page.surge.sh/

File: documentation-src/build.gulp.js [hunk 1]
```diff
@@ -36,58 +38,83 @@ var documentationRoot = path.join(projectRoot, 'documentation');
...
+
+var header = a.communityHeader({
+  menu:{
+    project: {
+      label: "algoliasearch-helper",
+      url: "/algoliasearch-helper-js/"
+    }
+  },
+  sideMenu: [
+    { name: "Getting started", dropdownItems: null, url: "/algoliasearch-helper-js/gettingstarted.html" },
```
Comment (bobylito) [inline documentation-src/build.gulp.js]: You should use relative path to the files. It will help local deployment (and not break on surge / staging)

Comment (JonasBa) [inline documentation-src/build.gulp.js]: Let me fix that.

**Comment by JonasBa** [comment]
For now we don't force the use of font inside the component, but we could provide base CSS that will provide the right font settings and that way we have it easily synced all over the place \o/

P.S: The whitespaces are different because you have a set `:root` font size to 14px.

**Comment by bobylito** [comment]
@JonasBa Can you fix the CSS of the header and add it to the PR?

File: documentation-src/build.gulp.js (hunk 1 above)
Comment (Haroenv) [inline documentation-src/build.gulp.js]: is there some way to have the local/dev server be `localhost:PORT/algoliasearch-helper` instead, that should be leading to a more similar experience in both cases

**Comment by Shipow** [comment]
+1 :)

**Comment by bobylito** [comment]
@JonasBa Can you work on the requested changes?

**Comment by bobylito** [comment]
The changesets of files is wrong.

**Pull request closed without merging**